pkg/statistics/handle/syncload/stats_syncload_test.go

```diff
@@ -211,17 +211,18 @@ func TestConcurrentLoadHistWithPanicAndFail(t *testing.T) {
 		task3, err3 := h.HandleOneTask(testKit.Session().(sessionctx.Context), task1, exitCh)
 		require.NoError(t, err3)
 		require.Nil(t, task3)
-
-		task, err3 := h.HandleOneTask(testKit.Session().(sessionctx.Context), nil, exitCh)
```

Comment:
Now, we merge the same request on the top. it has nothing in the task channel. so it will block.